README.md

```diff
@@ -1 +1,85 @@
-# post-it-app
\ No newline at end of file
+[![Build Status](https://travis-ci.org/nedemenang/post-it-app.svg?branch=develop)](https://travis-ci.org/nedemenang/post-it-app) [![Coverage Status](https://coveralls.io/repos/github/nedemenang/post-it-app/badge.svg?branch=develop)](https://coveralls.io/github/nedemenang/post-it-app?branch=develop)
+
+
+
+# PostIt Application
+
+
+# Background Information 
+
+PostIt is a simple application that allows friends and colleagues create groups for notifications. This way one person can post notifications to everyone by sending a message once. The application allows people create accounts, create groups and add registered users to the groups, and then send messages out to these groups whenever they want.
+
+## Features
+
+Users can sign up / log in to the Application
+
+Users can Create broad cast groups
+
+Users can add registered users to broadcast groups they have created
+
+
+### Why is this project useful
+
+It enables proper and easy dissemination of information by allowing users to broadcast messages to others users within a groups
+
+
+### Dependencies
+
+Node.js 
+
+babel-cli
+
+babel-core
+
+babel-eslint
+
+babel-loader
+
+babel-preset-es2015
+
+babel-watch
+
+body-parser
+
+express
+
+firebase
+
+eslint 
+
+### Installing and setup
+
+
+Navigate to directory of choice on terminal.
+
+Clone this repository on that directory.
+
+Ensure node is installed
+
+Install all app Dependencies
+
+Run the application using npm start 
+
+## Limitations of this project
+
+This project as at today is still in phase 1, therefore, you can only access the projects features using Postman.
+
+
+## Built With
+
+
+node.js
+
+
+## Authors
+
+* **Nnamso Edemenang** - *Post It App* - (https://github.com/nedemenang)
+
+## Heroku Link
+
+postitappnnam.herokuapp.com
+
+## License
+
+This project is licensed under the ISC License 
+
```

Comment:
Consider adding a 'Contributing' section to your README. This will give potential contributors information like how to contribute, what style guides the project follows, PR naming convention, etc.